.zsh-autosuggestionsrc

```diff
@@ -0,0 +1,13 @@
+# Setup zsh-autosuggestions
+source $HOME/.zsh-autosuggestions/autosuggestions.zsh
+
+# Enable autosuggestions automatically
+zle-line-init() {
+    zle autosuggest-start
+}
+
+zle -N zle-line-init
+
+# use ctrl+t to toggle autosuggestions(hopefully this wont be needed as
+# zsh-autosuggestions is designed to be unobtrusive)
+bindkey '^T' autosuggest-toggle
```

Comment:
@baykovr what about commenting this binding by default? It might override a user binding, and it's not actually needed anyway since autosuggestions is "made to work unobtrusively without it".